rotkehlchen/chain/evm/node_inquirer.py

```diff
@@ -531,34 +531,29 @@ def _query(self, method: Callable, call_order: Sequence[WeightedNode], **kwargs:
         for node_idx, weighted_node in enumerate(call_order):
             node_info = weighted_node.node_info
             web3node = self.web3_mapping.get(node_info, None)
-            if (
-                web3node is None and
-                node_info.name != self.etherscan_node_name and
-                node_info.name not in self.failed_to_connect_nodes
-            ):
-                success, _ = self.attempt_connect(node=node_info)
-                if success is False:
-                    self.failed_to_connect_nodes.add(node_info.name)
+            if web3node is None:
+                if node_info.name in self.failed_to_connect_nodes:
```

Comment:
is this a bugfix?

![download (45)](https://github.com/user-attachments/assets/81d5f66c-46a4-429b-bd85-7e59cb349277)